lib/otpbank.d.ts

```diff
@@ -13,22 +13,84 @@ declare namespace otpbank {
     consumerReceiptNeeded?: boolean;
     consumerRegistrationNeeded?: boolean;
     consumerRegistrationId?: string;
-  };
+  }
+
+  type TTransactionResult = 'success' | 'cardBlocked' | 'invalidCardNumber' | 'invalidCardBin' | 'cardExpired' |
+    'cardRestricted' | 'cardLost' | 'cardNotActive' | 'incorrectCardCredentials' | 'insufficientFunds' |
+    'cardDoesNotComplyWithIndustry' | 'cardUnknown' | 'cannotChargeCard' | 'cannotChargeCardWithGivenAmount' |
+    'error' | 'invalidAmount';
+
+  interface IGetTransactionReturn {
+    transactionId: string;
+    posId: string;
+    responseCode: string;
+    transactionResult: TTransactionResult;
```

Comment:
can't we emit TTransactionResult and use the enum here? `transactionResult: TransactionResults;`